### PR TITLE
quick fix for issue#73

### PR DESCRIPTION
**Pull request by calvera**
this is simple fix for #73

### CODE REVIEW

File: Builder/Admin/ListBuilder.php [hunk 1]
```diff
@@ -49,7 +49,11 @@ public function getFilterColumns()
 
     protected function findFilterColumns()
     {
-        foreach ($this->getAllFields() as $columnName) {
+        $columns = $this->getVariable('filters');
+        if (null == $columns || 0 == sizeof($columns)) {
+            $columns = $this->getDisplayColumns();
```
Comment (sescandell) [inline Builder/Admin/ListBuilder.php]: If we want to be able to filter on non displayed fields, you should better call `getAllFields()`

Could you please update this? And so we should be able to merge it.

Thank you!

Comment (loostro) [inline Builder/Admin/ListBuilder.php]: :+1: for merge, when this is updated

Comment (sescandell) [inline Builder/Admin/ListBuilder.php]: Finally not.

I'm working on this, and, I'm not sure this is enough.

I should push a PR including this change and some more. I think will can close that one

**Comment by loostro** [comment]
Closeing. Will be fixed in another PR by @sescandell

**Pull request closed without merging**